{
  "uid" : "f70cb4af97c4b5c3",
  "name" : "Delete user",
  "fullName" : "test.test_users.TestUsers#test_user_0",
  "historyId" : "2f5547d26dd0edb0cfaa459b2ff67a99",
  "time" : {
    "start" : 1749297799632,
    "stop" : 1749297801967,
    "duration" : 2335
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: {'code': 422, 'message': 'Users limit is reached: 20'}",
  "statusTrace" : "self = <test_users.TestUsers object at 0x7f8db819ba70>\n\n    @pytest.mark.smoke\n    @allure.title(\"Delete user\")\n    def test_user_0(self):\n>       user = self.users_api.create_user()\n\ntest/test_users.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/users/api.py:53: in create_user\n    model = self._validate_response(response, CreateUserResponseModel)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.users.api.UsersAPI object at 0x7f8db81dfb60>\nresponse = <Response [422]>\nmodel = <class 'services.users.models.model_create_user.CreateUserResponseModel'>\nstatus_code = 200, success = True\n\n    def _validate_response(self, response: requests.Response, model: type[BaseModel], status_code: int = 200,\n                          success: bool = True):\n        self.attach_response(response.json())\n        if success:\n>           assert response.status_code == status_code, response.json()\nE           AssertionError: {'code': 422, 'message': 'Users limit is reached: 20'}\n\nhelper/helper.py:20: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1749297799630,
      "stop" : 1749297799631,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_xunit_setup_method_fixture_TestUsers",
    "time" : {
      "start" : 1749297799631,
      "stop" : 1749297799631,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: {'code': 422, 'message': 'Users limit is reached: 20'}",
    "statusTrace" : "self = <test_users.TestUsers object at 0x7f8db819ba70>\n\n    @pytest.mark.smoke\n    @allure.title(\"Delete user\")\n    def test_user_0(self):\n>       user = self.users_api.create_user()\n\ntest/test_users.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nservices/users/api.py:53: in create_user\n    model = self._validate_response(response, CreateUserResponseModel)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <services.users.api.UsersAPI object at 0x7f8db81dfb60>\nresponse = <Response [422]>\nmodel = <class 'services.users.models.model_create_user.CreateUserResponseModel'>\nstatus_code = 200, success = True\n\n    def _validate_response(self, response: requests.Response, model: type[BaseModel], status_code: int = 200,\n                          success: bool = True):\n        self.attach_response(response.json())\n        if success:\n>           assert response.status_code == status_code, response.json()\nE           AssertionError: {'code': 422, 'message': 'Users limit is reached: 20'}\n\nhelper/helper.py:20: AssertionError",
    "steps" : [ {
      "name" : "Create user",
      "time" : {
        "start" : 1749297799632,
        "stop" : 1749297801966,
        "duration" : 2334
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: {'code': 422, 'message': 'Users limit is reached: 20'}\n",
      "statusTrace" : "  File \"/usr/local/lib/python3.12/site-packages/allure_commons/_allure.py\", line 202, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/usr/workspace/services/users/api.py\", line 53, in create_user\n    model = self._validate_response(response, CreateUserResponseModel)\n            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/workspace/helper/helper.py\", line 20, in _validate_response\n    assert response.status_code == status_code, response.json()\n",
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "e7b149a9d962ba7b",
        "name" : "API response",
        "source" : "e7b149a9d962ba7b.json",
        "type" : "application/json",
        "size" : 64
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "_xunit_setup_method_fixture_TestUsers::0",
    "time" : {
      "start" : 1749297802002,
      "stop" : 1749297802002,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test User"
  }, {
    "name" : "tag",
    "value" : "smoke"
  }, {
    "name" : "parentSuite",
    "value" : "test"
  }, {
    "name" : "suite",
    "value" : "test_users"
  }, {
    "name" : "subSuite",
    "value" : "TestUsers"
  }, {
    "name" : "host",
    "value" : "cb426ff55724"
  }, {
    "name" : "thread",
    "value" : "10-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_users"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 5,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 5
      },
      "items" : [ {
        "uid" : "1b53e0541737cef6",
        "status" : "failed",
        "statusDetails" : "AssertionError: {'code': 422, 'message': 'Users limit is reached: 20'}",
        "time" : {
          "start" : 1749297501288,
          "stop" : 1749297504317,
          "duration" : 3029
        }
      }, {
        "uid" : "e06b14cda35c8873",
        "status" : "failed",
        "statusDetails" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}",
        "time" : {
          "start" : 1749296700705,
          "stop" : 1749296702518,
          "duration" : 1813
        }
      }, {
        "uid" : "5363c84965eb9402",
        "status" : "failed",
        "statusDetails" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}",
        "time" : {
          "start" : 1749296327354,
          "stop" : 1749296329357,
          "duration" : 2003
        }
      }, {
        "uid" : "3ce61e079bc48abb",
        "status" : "failed",
        "statusDetails" : "AssertionError: {'code': 401, 'message': 'security requirements failed: authentication failed, please set correct \"Bearer\" header'}",
        "time" : {
          "start" : 1749295715104,
          "stop" : 1749295717620,
          "duration" : 2516
        }
      } ]
    },
    "tags" : [ "smoke" ]
  },
  "source" : "f70cb4af97c4b5c3.json",
  "parameterValues" : [ ]
}